### ISSUE TITLE
Persistent Volume Claim with storage class of 'default' is stuck in 'Pending' state

### ISSUE THREAD

**Issue by brandon-bethke-neudesic** [Azure/brigade#293]
Commit https://github.com/Azure/brigade/commit/81a81513d7b196d225ceed5118259d82ec19c0ad
introduced a user configurable storage class type for the Persistent Volume Claims. 

What is happening to me is that the PVC is getting stuck in the 'Pending' state and is requesting a storage class of 'default'. 
Prior to the commit, it seems that the storage class was left unspecified. Perhaps it should remain unspecified as the default rather than specifying a storage class of 'default'

```
brandonbethke:temp brandonbethke$ kubectl get pvc
NAME                                                 STATUS    VOLUME    CAPACITY   ACCESS MODES   STORAGECLASS   AGE
brigade-worker-01c5239j2spp4f3mb95w8rnrka-53780cf4   Pending                                       default        22m
```

**Comment by mumoshu** [comment]
@brandon-bethke-neudesic Great feedback!

Just confirming but you don't have a default storage class named `default` in your cluster, right?

I had been believing that it was conventional to have a PVC named/configured such, with , but sounds like not true at all after reading https://kubernetes.io/docs/concepts/storage/persistent-volumes/#class.

So,

> Perhaps it should remain unspecified as the default rather than specifying a storage class of 'default'

This fix makes sense to me!

**Comment by brandon-bethke-neudesic** [comment]
I do not have a storage class named 'default'. I was always under the impression that the default is 'standard'. But again, it's probably best to leave it unspecified by default.

```
brandonbethke:temp brandonbethke$ kubectl get storageclass
NAME                 PROVISIONER                AGE
standard (default)   k8s.io/minikube-hostpath   5d 
```

**Comment by brandon-bethke-neudesic** [comment]
Please see Pull Request https://github.com/Azure/brigade/pull/296

**Comment by technosophos** [comment]
Makes sense. I am reviewing the PR

**Comment by technosophos** [comment]
I believe this is closed with the merge of #296 and the release of 0.10
